index.html

```diff
@@ -679,18 +679,18 @@
 const writer = new NFCWriter();
 const reader = new NFCReader();
 
-reader.onreading = message => {
-  if (message.records[0].recordType == 'empty') {
+reader.onreading = event => {
+  if (event.message.records[0].recordType == 'empty') {
```

Comment:
I think it will look nicer to create a local variable.

const message = event.message;

Reply:
At least in this case